confgenerator/config/config.go

```diff
@@ -521,10 +510,19 @@ var (
 				return nil
 			},
 		},
-		"hostmetrics":     collectionIntervalValidation,
-		"iis":             collectionIntervalValidation,
-		"mssql":           collectionIntervalValidation,
-		"exclude_metrics": metricPrefixesValidation,
+		"hostmetrics": collectionIntervalValidation,
+		"iis":         collectionIntervalValidation,
+		"mssql":       collectionIntervalValidation,
+		"exclude_metrics": map[string]func(interface{}) error{
+			"metric_prefixes": func(v interface{}) error {
+				for _, prefix := range v.([]string) {
+					if !strings.HasSuffix(prefix, "/") {
+						return fmt.Errorf(`must end with "/"`)
```

Comment:
Put the prefix in the error message, such as

```
fmt.Errorf(`prefix %q must end with "/"`, prefix)
```

Reply:
See error msgs in:
```
sophieyfang@sophieyfang:~/tmp/ops-agent$ cat confgenerator/testdata/invalid/linux/metrics-processor_unsupported_type/golden_error
metrics processor "processor_1" with type "unsupported_type" is not supported. Supported metrics processor types: [exclude_metrics].
```

Reply:
Okay, that doesn't say which list item but it's not terrible.

Reply:
Changed to
```
sophieyfang@sophieyfang:~/tmp/ops-agent$ cat confgenerator/testdata/invalid/linux/metrics-processor_exclude_metrics_invalid_parameter_metric_prefixes/golden_error
parameter "metric_prefixes" in "exclude_metrics" type metrics processor "metrics_filter" has invalid value ["agent.googleapis.com/processes/not_end_with_forward_slash_with_star" "not.agent.googleapis.com/processes/*"]: agent.googleapis.com/processes/not_end_with_forward_slash_with_star must end with "/*" | not.agent.googleapis.com/processes/* must start with "agent.googleapis.com/"
```